gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/management/impl/InternalManagementService.java

```diff
@@ -408,7 +411,20 @@ private void handleTableCreate(Object[] data) {
     this.tableMBeanDataUpdater.addUpdatable(tableMBeanBridge.getFullName(), tableMBeanBridge); // start updates after creation
   }  
 
-  List<String> getTableDefinition (String parentSchema, String tableName){   
+  private boolean hasInternalTableSchema(GemFireContainer container) {
+    for (String schema : CallbackFactoryProvider.getStoreCallbacks().getInternalTableSchema()) {
+      if (schema.equalsIgnoreCase(container.getSchemaName())) {
+        return true;
+      }
+    }
+    // This constant is defined at snappy-store side, so won't be included in above response from StoreCallback.
+    if (StoreCallbacks.INTERNAL_SCHEMA_NAME.equalsIgnoreCase(container.getSchemaName())) {
+      return true;
```

Comment:
>> Why not add this to the list returned by CallbackFactoryProvider.getStoreCallbacks()? Will be cleaner.
That was because INTERNAL_SCHEMA_NAME is defined in snappy-store while the callback is implemented in snappy-tools.

Now as I look at it, I see there are two constants defined for the same string, one each in snappy-store and snappy-tools (StoreCallbacks.INTERNAL_SCHEMA_NAME and ColumnFormatRelation.INTERNAL_SCHEMA_NAME). May be I can merge these into one and access it via callbacks.